Sources/POI/OAPOI.mm

```diff
@@ -329,6 +329,18 @@ - (NSString *)getDescription:(NSString *)lang
     return res;
 }
 
+- (NSString *)getAdditionalInfo:(NSString *)key
+{
+    if (!_values)
```

Comment:
It's not clear what's happening here. Wouldn't simply returning` return [_values objectForKey:key]; work?`